h/views/api/groups.py

```diff
@@ -132,6 +135,7 @@ def update(group, request):
     route_name="api.group_upsert",
     request_method="PUT",
     permission="upsert",
+    link_name="group.replace",
```

Comment:
I wonder if `replace` is the best name here. If the group already exists then metadata just gets updated. "replace" might imply that the existing group and associated resources get deleted. It is a little verbose but perhaps `create_or_update`?